src/internal/pjsdb/queue_crud_test.go

```diff
@@ -0,0 +1,41 @@
+package pjsdb_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pjsdb"
+	"github.com/pachyderm/pachyderm/v2/src/internal/require"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage/fileset"
+)
+
+func TestForEachQueue(t *testing.T) {
+	t.Run("valid", func(t *testing.T) {
+		withDB(t, func(ctx context.Context, t *testing.T, db *pachsql.DB) {
+			withFilesetStorage(t, ctx, db, func(ctx context.Context, s *fileset.Storage) {
+				withTx(t, ctx, db, s, func(ctx context.Context, tx *pachsql.Tx, s *fileset.Storage) {
+					d := dependencies{ctx: ctx, tx: tx, s: s}
+					var err error
+					targetFs := mockFileset(t, d, "/spec", "#!/bin/bash; echo 'hello';")
```

Comment:
Why `/bin/bash` instead of `/bin/sh`?

Reply:
The contents of the string don't matter here. I wrote something that looked real so it would be easier to understand what each fileset does. In this case, the spec is a fileset that contains the code that will be ran within a job.